src/routers/editorRouter.js

```diff
@@ -20,14 +20,15 @@ import {
   putGenericWord,
   getGenericWord,
 } from '../controllers/genericWords';
+import authorization from '../middleware/authorization';
 
 const editorRouter = express.Router();
 
 /* These routes are used to allow users to suggest new words and examples */
-editorRouter.post('/words', mergeWord);
-editorRouter.put('/words/:id', putWord);
-editorRouter.post('/examples', mergeExample);
-editorRouter.put('/examples/:id', putExample);
+editorRouter.post('/words', authorization(['merger', 'admin']), mergeWord);
```

Comment:
Only mergers and admins of the platform can merge words and examples into the database.